Pipfile

```diff
@@ -32,9 +32,9 @@ pydantic = "*"
 uvloop = "*"
 httptools = "*"
 "ga4gh.vrs" = {version = "==0.7.0rc3", extras = ["extras"]}
-gene-normalizer = ">=0.1.21"
+gene-normalizer = "==0.1.21"
```

Comment:
I'm guessing this is changing in like #144 ?

Reply:
Yes!